src/store/inventory-actions.js

```diff
@@ -249,3 +250,18 @@ export const toggleDrawer = (isOpened) => ({
   type: TOGGLE_DRAWER,
   payload: { isOpened },
 });
+
```

Comment:
The mocks call for an item Downloading export. 
The code for this isnt here. Im assuming because the export works fast enough that you dont really need it. Does UX want 2 notifications to pop up or was this set up fine/agreed on? 

![Screenshot 2024-06-27 at 1 19 10 PM](https://github.com/RedHatInsights/insights-inventory-frontend/assets/60629070/b509b286-2147-4bc7-bf0e-a812cd7969f5)

Reply:
@adonispuente The Download notification will be issued by the Export service.